pages/common/croc.md

```diff
@@ -0,0 +1,32 @@
+# croc
+
+> Send and recieve files easily and security over any network.
+> More information: <https://github.com/schollz/croc>.
+
+- Send a file or folder:
+
+`croc send {{path/to/file}}`
+
+- Send a file or folder with a specific passphrase:
+
+`croc send --code {{your-own-created-passphrase}} {{path/to/file}}`
+
+- Recieve a file or folder on reciving machine:
+
+`croc {{the-passphrase-to-recieve-a-file}}`
+
+- Send over a custom relay and custom relay:
+
+`croc --relay {{ip-to-relay}} send {{path/to/file}}`
+
+- Recieve over a custom relay and custom relay:
+
+`croc --relay {{ip-to-relay}} {{the-passphrase-to-recieve-a-file}}`
+
+- Host your own croc relay with default ports ("`9009,9010,9011,9012,9013`"):
+
+`croc relay`
+
+- Host your own croc relay with custom ports:
+
+`croc relay --ports {{optional-relay-ports}}`
```

Comment:
```suggestion
`croc relay --ports {{optional_relay_ports}}`
```

Reply:
Think it might actually just not be included in the `tldr` as it is quite 'advanced' and one can read that if needed through the `--help` command.

But I will add bout that `--help` flag!

Reply:
Removed the custom ports part, and added about the `--help` flag instead. As that command is better and probably more helpful than this advanced feature.